Mathlib/RingTheory/Flat.lean

```diff
@@ -93,6 +93,9 @@ instance self (R : Type u) [CommRing R] : Flat R R :=
 
 variable (M : Type v) [AddCommGroup M] [Module R M]
 
+/-- An `R`-module `M` is flat iff for all finitely generated ideals `I` of `R`, the
+tensor product of the inclusion `I → R` and the identity `M → M` is injective. See
+`iff_rTensor_injective'` to extend to all ideals `I`. --/
 lemma iff_rTensor_injective :
     Flat R M ↔ (∀ ⦃I : Ideal R⦄ (_ : I.FG), Injective (rTensor M I.subtype)) := by
   have aux : ∀ (I : Ideal R), ((TensorProduct.lid R M).comp (rTensor M I.subtype)) =
```

Comment:
might remove some parentheses here if you want
```suggestion
    Flat R M ↔ ∀ ⦃I : Ideal R⦄, I.FG → Injective (rTensor M I.subtype) := by
  have aux : ∀ I : Ideal R, (TensorProduct.lid R M).comp (rTensor M I.subtype) =
```
(there's one more below after the =)

Reply:
Ah, this part is not my code. See #7567. I should probably not touch it. I'll undo this.

Reply:
Reverted.

Reply:
I think you need not abstain from touching existing code as long as you see an improvement, but LGTM

maintainer merge